### ISSUE TITLE
[css-anchor-position-1] position-try-options interaction with position-visibility

### ISSUE THREAD

**Issue by kbrilla** [w3c/csswg-drafts#10341]
When exploring chrome blog describing https://developer.chrome.com/blog/anchor-positioning-api?hl=en i encountered that there is no support for using  `position-visibility` with `position-try-options` or at least it does not work as one would expect.

For example:
```css
  position-visibility: no-overflow;
  position-try-options: flip-block;
```
as I scroll I would expect 
first flip to happen as soon as overflow would kick in then as overflow would kick in again for tooltip to be hidden.
so:
<img width="436" alt="image" src="https://github.com/w3c/csswg-drafts/assets/22288449/88e9587b-1aaa-497c-9c90-75763e23e2cb">
<img width="464" alt="image" src="https://github.com/w3c/csswg-drafts/assets/22288449/81f6a243-5e27-4900-8b81-ced077e1e8be">
And now it should be hidden but stay visible
<img width="439" alt="image" src="https://github.com/w3c/csswg-drafts/assets/22288449/ce903ae1-5bdf-4b2b-b231-fbdd58f0c820">
https://codepen.io/krzbri/pen/OJYymRv

and now 
```css
  position-visibility: no-overflow;
  position-try-options: flip-block;
```
here I would expect basically the same same behaviour with the exception that now we are talking about visibility of anchor. 
What happens though is  that as soon as top anchor is hidden it does not try to first try to use bottom anchor as  `  position-try-options: flip-block;` would suggest.  
<img width="464" alt="image" src="https://github.com/w3c/csswg-drafts/assets/22288449/f5523504-ee3e-4bd2-ac2e-7e72b91f4629">
So here I would except to first switch to bottom anchor and only when this one is hidden for tooltip to be hidden. 

https://codepen.io/krzbri/pen/jOobmmd

**Comment by fantasai** [comment]
@kbrilla I think this is already what the spec requires? Maybe would be useful to add a WPT testcase, but if I'm understanding the request correctly I think we don't need any spec edits.

The idea of `no-overflow` is that

1. We try positioning on top, when that overflows
2. Try positioning on the bottom, but
3. If both options would overflow, hide the item

Lmk if that matches, or if you're asking for something different.

**Comment by fantasai** [comment]
Actually, looking more closely at the testcase, I think this is caused by
- https://github.com/w3c/csswg-drafts/issues/12607

**Comment by fantasai** [comment]
The problem is that both the anchor and the anchored tooltip are inside the scrollable containing block, and everything fits inside the scroller so no fallback behavior is triggered. What's wanted is to use the scrollport.

**Comment by tabatkins** [comment]
Agreed, this is exactly the problem. The abspos is (correctly, per spec) using the scrollable CB, which is the whole scrollable area, so it never overflow. It *wants* to be using the scrollport, so that's #12607.
